enterprise/server/remote_execution/cgroup/BUILD

```diff
@@ -5,11 +5,15 @@ go_library(
     srcs = ["cgroup.go"],
     importpath = "github.com/buildbuddy-io/buildbuddy/enterprise/server/remote_execution/cgroup",
     visibility = ["//visibility:public"],
-    deps = [
-        "//proto:remote_execution_go_proto",
-        "//server/util/log",
-        "//server/util/status",
-    ],
+    deps = select({
```

Comment:
same thing about using compatible_with here

Reply:
gazelle added this diff after I added the `//go:build` directive. gopls complains without that directive, because it's trying to build on all platforms. I added `target_compatible_with` in addition, so that this target doesn't get built on windows